sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_helpers.py

```diff
@@ -0,0 +1,41 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+import json
+import zlib
+
+MAX_CHUNK_SIZE = 1000000
+CHAR_SIZE = 4
+
+
+def _split_chunks(logs):
+    chunks = []
+    chunk_size = 0
+    curr_chunk = []
+    for log in logs:
+        # each char is 4 bytes
+        size = len(json.dumps(log)) * CHAR_SIZE
+        if chunk_size + size < MAX_CHUNK_SIZE:
+            curr_chunk.append(log)
+            chunk_size += size
+        else:
+            chunks.append(curr_chunk)
+            curr_chunk = [log]
+            chunk_size = 0
```

Comment:
This should be set to the current log's size 
```suggestion
            chunk_size = size
```